### PR TITLE
Fixes navbar on mobile

### PR DESCRIPTION
**Pull request by joaojusto**
Old:

screen shot 2015-08-28 at 16 58 31

New:

screen shot 2015-08-28 at 16 58 44

### CODE REVIEW

File: source/partials/_nav.slim [hunk 1]
```diff
@@ -5,19 +5,20 @@
...
   = nav_overlay_item('Blog', 'https://blog.subvisual.co/', 'blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
+header#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
+  = link_to '/', "aria-label" => "Homepage", class: 'Nav-logo'
     = partial 'nav/logo'
 
-  a.Nav-menu href="#"
+  .Nav-menu
```
Comment (gabrielpoca) [inline source/partials/_nav.slim]: Why are you removing the `a`? Is this element going to be focusable?

Comment (joaojusto) [inline source/partials/_nav.slim]: Yes it is supposed to be focusable. My bad.

**Comment by joaojusto** [comment]
@gabrielpoca bump

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**